Source/Shaders/ShadowVolumeFS.glsl

```diff
@@ -0,0 +1,23 @@
+#ifdef GL_EXT_frag_depth
+#extension GL_EXT_frag_depth : enable
+#endif
+
+varying vec4 v_color;
+
+#ifdef GL_EXT_frag_depth
+// emulated noperspective
+varying float v_WindowZ;
+#endif
+
+void writeDepthClampedToFarPlane()
+{
+#ifdef GL_EXT_frag_depth
```

Comment:
Do we need this `#ifdef` here and in the vertex shader?  Should we even compile the shader if the extension is not supported?